src/cxx.cc

```diff
@@ -65,7 +65,11 @@ static void initString(String *self, const char *s, size_t len) {
   }
 }
 
+#ifdef CXXBRIDGE_HAS_STRING_VIEW
+String::String(std::string_view sv) : String(sv.data(), sv.length()) {}
```

Comment:
Forwarding it to ptr/len ctor that does the nice asserts.

FYI, empty string_view has nullptr and 0 size.